Jenkinsfile

```diff
@@ -102,6 +102,10 @@ try {
         }
         parallel parallel_containers
 
+        if (env.BRANCH_NAME == 'master') {
+          build job: 'test-shiny-server-os', wait: false
```

Comment:
this needs to be the full project name. For https://buildmaster.rstudioservices.com/job/shiny/job/test-shiny-server-os/ that would be `shiny/test-shiny-server-os`.

Reply:
thanks, yeah wasn't sure exactly what path it would need.